lib/csv_loadable.rb

```diff
@@ -1,6 +1,6 @@
 require 'CSV'
 
-module CsvLoadable
+class CsvLoadable
 
```

Comment:
I'd like to get KK's input on this.. She introduced the module and I'm curious what the consequences of changes this to a class are?? I have recently been using the module to help write the manager classes

Reply:
That was a Bob change made during our Group Project collab time today. That's part of why the stub runs.